cloud/amazon/ec2.py

```diff
@@ -613,6 +620,19 @@ def get_instance_info(inst):
         instance_info['ebs_optimized'] = False
 
     try:
+        bdm_dict = {}
```

Comment:
Indeed you are right, i should have to open another merge request instead to insert this here.
But this is ONLY an extra output for some info of any attached disk.

Do I have to separate the requests ?